includes/sdk-debuggability/browser.md

```diff
@@ -0,0 +1,68 @@
+#### How to debug
+
+Debugging in a browser can help you identify problems related to your code's implementation, as well as potential issues within the SDKs you're using. Here's a basic guide on how to use the browser's built-in Developer Tools (DevTools) for debugging.
+
+##### Console
+
+You can find JavaScript errors under **Inspect > Console**, which might have the details about the line of code and file that caused the problem. The console also allows you to execute JavaScript code in real-time.
+
+* Enable debug mode by following these [instructions](./#debug-mode). Then With the default logger, extra function context information will be output to the developer console when any SDK public method is invoked, which can be helpful for debugging.
+
+* If you're able to send the event successfully after entering `amplitude.init(API_KEY, 'USER_ID')` in the browser console, it indicates that your `amplitude.init` call might not have been triggered. We support deferred initialization, so the events will be dispatched after the initialization call. Therefore, please check your implementation.
```

Comment:
> If you're able to send the event successfully after entering `amplitude.init(API_KEY, 'USER_ID')` in the browser console, it indicates that your `amplitude.init` call might not have been triggered. .... please check your implementation.

I am confused here. When and why do we need to check if an amplitude instance have already been initialized? If the customer is using deferred initialization, the events can still been sent later.

Reply:
some customers don't know we support deferred initialization.
They thought they have init and track call, and they thought the event should be triggered but there is no event has been ingested. 
There are 2 cases, I added more info in this part. 
1. they can have an init call with a wrong instance or wrong instance name. so their init won't ever work.
2. the init doesn't been triggered as expected.

Reply:
Can we reword it like

"Amplitude supports SDK deferred initialization. Events tracked before initialization will be dispatched after the initialization call. If you cannot send events but are able to send the event successfully after entering `amplitude.init(API_KEY, 'USER_ID')` in the browser console, it indicates that your `amplitude.init` call might not have been triggered in your codebase. Therefore, please check your implementation."